src/OrbitQt/MoveFilesDialog.cpp

```diff
@@ -33,4 +33,20 @@ void MoveFilesDialog::EnableCloseButton() {
   ui_->closeButton->setEnabled(true);
 }
 
+void MoveFilesDialog::closeEvent(QCloseEvent* event) {
+  if (ui_->closeButton->isEnabled()) {
+    event->accept();
+  } else {
+    event->ignore();
+  }
+}
```

Comment:
does it prevent close on escape?

Reply:
Nope, that's the reason for `keyPressEvent` below (as you have seen), but we still need this.